apps/web/src/actions/user-profile-mod.ts

```diff
@@ -37,43 +37,58 @@ export const modifyRegistrationData = authenticatedAction
 				personalWebsite,
 				phoneNumber,
 				countryOfResidence,
+				uploadedFile,
 			},
 			ctx: { userId },
 		}) => {
 			const user = await getUser(userId);
```

Comment:
I see the logic here, but these are all update calls that will not do anything if the clerk id doesn't exist so we do not lose anything here. I think you can remove this db call.